src/main/daml/ContingentClaims/Core/Claim.daml

```diff
@@ -83,8 +83,8 @@ when : Inequality t x o -> Claim t x a o -> Claim t x a o
 when = When
 
 -- | Smart constructor for `Anytime`.
-anytime : Inequality t x o -> Claim t x a o -> Claim t x a o
-anytime = Anytime
+anytime : Inequality t x o -> Text -> Claim t x a o -> Claim t x a o
```

Comment:
I haven't changed the `anytime` smart constructor though to use `(Text, Claim)` as input